.github/CODEOWNERS

```diff
@@ -5,5 +5,4 @@
 # the repo. Unless a later match takes precedence,
 # @global-owner1 and @global-owner2 will be requested for
 # review when someone opens a pull request.
-*       @haraldh @danimo @johannbg
-
+*       @haraldh @laszlogombos @Conan-Kudo
```

Comment:
Since `@haraldh` isn't a member of the organization or project, this is considered invalid.

Reply:
❤️